typedbSchemaBuilder/Builder.py

```diff
@@ -57,25 +57,25 @@ def abstract(self, type_: str, qid: int = -1):
             self._context = type_
             self._schema += "\n" + type_ + " abstract;"
 
-        query_log = copy.deepcopy(self._query_log)
+        constraint_log = copy.deepcopy(self._constraint_log)
         if qid == -1:
```

Comment:
`qid` here is short for "Query ID" - as such, this PR should also change this (and other similar occurences - there are further instances of `qid` but other things in the same vein would include, for example, changing `for q in self._constraint_log`) to a more appropriate name.